cmd/test/main_test.go

```diff
@@ -918,6 +919,12 @@ func test_testCommand_no_record_flag(t *testing.T, ctx *TestCommandContext) {
 	assert.Equal(t, ViolationsFoundError, err)
 }
 
+func test_testCommand_save_results_flag(t *testing.T, ctx *TestCommandContext) {
+	flags := TestCommandFlags{SaveResults: "./"}
+	err := flags.Validate()
+	assert.NoError(t, err)
```

Comment:
**Optional** - You could add a test case for when there is a wrong path and an error is thrown